src/pages/Filing/FilingApp/FileDetailsValidation.tsx

```diff
@@ -6,27 +6,32 @@ import { fileSubmissionValidationStatus } from './FileSubmission.data';
 interface FileDetailsProperties {
   dataGetSubmissionLatest: SubmissionResponse | undefined;
   errorGetSubmissionLatest: unknown;
+  hasWarnings: boolean;
 }
 
 function FileDetailsValidation({
   dataGetSubmissionLatest,
   errorGetSubmissionLatest,
+  hasWarnings,
 }: FileDetailsProperties): JSX.Element | null {
   // Should only show once an validation has completed
   if (
-    !dataGetSubmissionLatest?.filename &&
-    !dataGetSubmissionLatest?.submission_time
+    errorGetSubmissionLatest ||
+    (!dataGetSubmissionLatest?.filename &&
+      !dataGetSubmissionLatest?.submission_time)
   )
     return null;
 
   // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-  const validationStatusMessage: string = errorGetSubmissionLatest
-    ? fileSubmissionValidationStatus[FileSubmissionState.VALIDATION_ERROR]
-    : // @ts-expect-error key in
-      dataGetSubmissionLatest.state in fileSubmissionValidationStatus
-      ? // @ts-expect-error use key
-        fileSubmissionValidationStatus[dataGetSubmissionLatest.state]
-      : '';
+  const validationStatusMessage: string =
+    hasWarnings &&
+    dataGetSubmissionLatest.state === FileSubmissionState.SUBMISSION_ACCEPTED
+      ? 'Your file contains no errors and warnings have been verified'
+      : // @ts-expect-error key in
+        dataGetSubmissionLatest.state in fileSubmissionValidationStatus
```

Comment:
Oh interesting... I added the exception to [this ticket](https://github.com/cfpb/sbl-frontend/issues/528)